File: comms/email/delete-email/delete-email.py
from twilio import TwilioComms

client = TwilioComms(
    account_id="<username>",
    auth_token="<password>",
)

client.emails.delete(
    email_id="comms_email_01h9krwprkeee8fzqspvwy6nq8",
)

File: comms/email/delete-email/delete-email.php
<?php

namespace Example;

use Twilio\TwilioComms;

$client = new TwilioComms(
    accountId: '<username>',
    authToken: '<password>',
);
$client->emails->delete(
    'comms_email_01h9krwprkeee8fzqspvwy6nq8',
);
